docs/introduction/Examples.md

```diff
@@ -1 +1,137 @@
-## Examples
\ No newline at end of file
+## Examples
+### Simple - reading from an existing xlsx spread sheet.
+The following C plus plus code will read the values from an xlsx file and print the string values to the screen. This is a very simple example to get you started.
+
+```
+#include <iostream>
+#include <xlnt/xlnt.hpp>
+
+using namespace std;
+using namespace xlnt;
```

Comment:
I know it saves some typing and it's generally safe in non-header files, but I'd still prefer to avoid "using namespace ...;" whenever possible at least in xlnt and its documentation.